daemon/src/maker_cfd.rs

```diff
@@ -521,7 +521,18 @@ impl Actor {
             }
         };
 
-        // 2. Insert CFD in DB
+        // 2. check if order has acceptable amounts
+        if quantity.0 < current_order.min_quantity.0 || quantity.0 > current_order.max_quantity.0 {
```

Comment:
Instead of reaching into new types via `.0`, it would be better if we could establish a habit of implementing the required traits on it ...